Update Turbopack docs for beta release (#4769)
vercel/turbo: docs/pages/pack/docs/features/frameworks.mdx

## Next

> To begin with, Turbopack has been focused on providing a great experience for Next.js's dev server. We're using this as our initial goal to show what Turbopack can do. In the future, we want Turbopack to act as a low-level engine for other frameworks.

Reviewer: ```suggestion
To begin with, Turbopack is focused on providing a great experience for Next.js's dev server. We're using this as our initial goal to show what Turbopack can do. In the future, we want Turbopack to act as a low-level engine for other frameworks.
```
